Firestore/Source/Core/FSTFirestoreClient.h

```diff
@@ -109,12 +110,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 /** Write mutations. completion will be notified when it's written to the backend. */
 - (void)writeMutations:(std::vector<FSTMutation *> &&)mutations
-            completion:(nullable FSTVoidErrorBlock)completion;
+            completion:(util::StatusCallback)completion;
```

Comment:
Is it worth renaming "completion" to "callback" in the places that take `util::StatusCallback` ?

Reply:
Sure. I was trying to make this a more minimal change, but I've done so.

I've also renamed `TransactionCompletion` to `TransactionResultCallback`.